src/mergeJSDOMOptions.js

```diff
@@ -0,0 +1,21 @@
+const isFunction = (value) => typeof value === 'function';
+
+export default (defaultOptions, customOptions = {}) => {
+  const result = { ...defaultOptions };
+
+  Object.entries(customOptions).forEach(([customOptionKey, customOptionValue]) => {
+    const defaultOptionValue = result[customOptionKey];
+    const areBothFunctions = [defaultOptionValue, customOptionValue].every(
+      isFunction,
+    );
+
+    result[customOptionKey] = areBothFunctions
+      ? (...args) => {
+          defaultOptionValue(...args);
+          customOptionValue(...args);
```

Comment:
Thanks for making custom options apply last. This makes it possible to override things set by the default `beforeParse` function.